packages/code-du-travail-api/src/server/routes/items/getRelatedItems.js

```diff
@@ -126,12 +123,11 @@ async function getSearchBasedItems({ title, settings }) {
   );
 }
 
-// get related items, depending on : covisits present & non empty and A/B testing ratio
+// get related items, depending on : covisits present & non empty
 async function getRelatedItems({ title, settings, slug, covisits }) {
-  const useCovisits = Math.random() < abRatio;
-
-  const covisitedItems =
-    covisits && useCovisits ? await getCovisitedItems({ covisits, slug }) : [];
+  const covisitedItems = covisits
+    ? await getCovisitedItems({ covisits, slug })
+    : [];
```

Comment:
should we add documents (from sem or Es) when covisits don't yet exist ?

Reply:
yep, we still use search : 
https://github.com/SocialGouv/code-du-travail-numerique/blob/cc2972b987ab73a23fce02cd8b98d425d16f0976/packages/code-du-travail-api/src/server/routes/items/getRelatedItems.js#L135